src/libcore/ops/arith.rs

```diff
@@ -537,6 +537,20 @@ rem_impl_integer! { usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 }
 
 macro_rules! rem_impl_float {
     ($($t:ty)*) => ($(
+
+        /// The remainder from the division of two floats.
+        ///
+        /// The remainder has the same sign as the dividend and is computed as:
+        /// `x - (x / y).trunc() * y`.
+        ///
+        /// # Examples
+        /// ```
+        /// let x: f32 = 4.0;
+        /// let y: f32 = 2.5;
+        /// let remainder = x - (x / y).trunc() * y;
```

Comment:
I'd comment the expected value (1.5) for demonstration purposes. Might be nicer to choose inputs with a greater quotient too, so this doesn't look like just subtraction.

Reply:
Alright! So what about 15.2 and 2.3? (Just some random numbers). Then I would also add a comment that the result is: 1.4

Reply:
Random is fine, but maybe stick to base-2 numbers that float can represent exactly.

Reply:
@cuviper I updated the example! What about this?

Reply:
Works for me!